doc/FAQ.md

```diff
@@ -94,26 +94,25 @@ in German:
 When a certain language is forced, the language remains until session is closed.
 
 <a name="contacts"></a>
-### What is the difference between blocked|ignored|archived|hidden contacts?
+### How do blocked, ignored, archived and hidden contacts behave?
 
-We prevent direct communication with **blocked contacts**.
-They are not included in delivery, and their own posts to you are not imported.
-However their conversations with your friends will still be visible in your stream.
-If you remove a contact completely, they can send you another friend request.
-Blocked contacts cannot do this. They cannot communicate with you directly, only through friends.
+##### Blocked
 
-**Ignored contacts** are included in delivery - they will receive your posts and private messages.
-However we do not import their posts or private messages to you.
-Like blocking, you will still see this person's comments to posts made by your friends.
+Direct communication will be blocked. Blocked contacts are not included in delivery, and their own posts to you are not imported. However their conversations with your friends will still be visible in your stream. If you remove a contact completely, they can send you another friend request. Blocked contacts cannot do this. They cannot communicate with you directly, only through friends.
```

Comment:
Please add a new line after each sentence. It doesn't affect the display but makes this file easier to read and change. This goes for all Markdown files.